### ISSUE TITLE
python3.8: Sphinx docs generation, ast.py related error

### ISSUE THREAD

**Issue by osuchw** [altair-viz/altair#1781]
ast.py changes in Python3.8 broke Sphinx documentation generator resulting in _SyntaxError_.
```
# Sphinx version: 2.2.1
# Python version: 3.8.0 (CPython)
# Docutils version: 0.15.2 release
# Jinja2 version: 2.10.3
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "c:\py\python38\lib\site-packages\sphinx\cmd\build.py", line 272, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "c:\py\python38\lib\site-packages\sphinx\application.py", line 269, in __init__
    self._init_builder()
  File "c:\py\python38\lib\site-packages\sphinx\application.py", line 330, in _init_builder
    self.events.emit('builder-inited')
  File "c:\py\python38\lib\site-packages\sphinx\events.py", line 103, in emit
    results.append(callback(self.app, *args))
  File "c:\src\altair\altair\sphinxext\altairgallery.py", line 268, in main
    save_example_pngs(examples, image_dir)
  File "c:\src\altair\altair\sphinxext\altairgallery.py", line 130, in save_example_pngs
    chart = eval_block(example['code'])
  File "c:\src\altair\altair\utils\execeval.py", line 38, in eval_block
    tree = ast.parse(code, filename='<ast>', mode='exec')
  File "c:\py\python38\lib\ast.py", line 47, in parse
    return compile(source, filename, mode, flags,
  File "<ast>", line 1
    Anscombe's Quartet
                     ^
SyntaxError: EOL while scanning string literal
```
A patch with a possible fix attached.

[py38-docgen.patch.txt](https://github.com/altair-viz/altair/files/3828920/py38-docgen.patch.txt)

**Comment by jakevdp** [comment]
Thanks! Would you be willing to open a pull request with your change?

**Comment by osuchw** [comment]
Pull request #1782 created.

**Comment by jakevdp** [comment]
Closed by #1782
